### PR TITLE
i18n(it): translate `guides/sidebar`

### PR DESCRIPTION
**Pull request by itsmatteomanf**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

Creates the Italian translation for `guides/sidebar.mdx`.

### CODE REVIEW

File: docs/src/content/docs/it/guides/sidebar.mdx [hunk 1]
```diff
@@ -0,0 +1,480 @@
+---
+title: Navigazione nella barra laterale
+description: scopri come impostare e personalizzare i collegamenti di navigazione della barra laterale del tuo sito Starlight.
```
Comment (ematipico) [inline docs/src/content/docs/it/guides/sidebar.mdx:3]:
```suggestion
description: Scopri come impostare e personalizzare i collegamenti di navigazione della barra laterale del tuo sito Starlight.
```

(not sure if it should be capitalized or not)

File: docs/src/content/docs/it/guides/sidebar.mdx [hunk 2]
```diff
@@ -0,0 +1,480 @@
...
+	]}
+/>
+
+### Gruppi
+
+Puoi aggiungere struttura alla barra laterale raggruppando i collegamenti correlati sotto un'intestazione comprimibile.
+I gruppi possono contenere sia collegamenti che altri sottogruppi.
+
+Aggiungi un gruppo utilizzando un oggetto con le proprietà `label` e `items`.
+Il `label` verrà utilizzato come intestazione del gruppo.
```
Comment (ematipico) [inline docs/src/content/docs/it/guides/sidebar.mdx:90]:
```suggestion
`label` verrà utilizzato come intestazione del gruppo.
```

Articles for english words in Italian are subjective. If we can avoid them, we can create less bikeshedding.

File: docs/src/content/docs/it/guides/sidebar.mdx [hunk 3]
```diff
@@ -0,0 +1,480 @@
...
+	]}
+/>
+
+### Gruppi generati automaticamente
+
+Starlight può generare automaticamente un gruppo nella barra laterale in base a una directory dei tuoi documenti.
+Ciò è utile quando non desideri inserire manualmente ciascun elemento della barra laterale in un gruppo.
+Per impostazione predefinita, le pagine verranno ordinate alfabeticamente in base al nome del file.
+
+Aggiungi un gruppo generato automaticamente utilizzando un oggetto con le proprietà `label` e `autogenerate`. La tua configurazione di `autogenerate` deve specificare la `directory` da utilizzare per le voci della barra laterale. Ad esempio, con la seguente configurazione:
```
Comment (ematipico) [inline docs/src/content/docs/it/guides/sidebar.mdx:145]:
```suggestion
Aggiungi un gruppo generato automaticamente utilizzando un oggetto con le proprietà `label` e `autogenerate`. La tua configurazione di `autogenerate` deve specificare quale `directory` da utilizzare per le voci della barra laterale. Ad esempio, con la seguente configurazione:
```

**Review by ematipico — APPROVED** [review]

File: docs/src/content/docs/it/guides/sidebar.mdx (hunk 1 above)
Comment (itsmatteomanf) [reply docs/src/content/docs/it/guides/sidebar.mdx:3]: it is in English. Will change.

File: docs/src/content/docs/it/guides/sidebar.mdx (hunk 2 above)
Comment (itsmatteomanf) [reply docs/src/content/docs/it/guides/sidebar.mdx:90]: To me it sounds slightly better with the article in front... without it sounds like `label` itself is used as the heading, and the value it assumes.

Comment (ematipico) [reply docs/src/content/docs/it/guides/sidebar.mdx:90]: I know, but I've seen people using feminine or masculine articles for the same words. E.g. la repository vs il repository 

Unless we stick to something (written somewhere), we will end up with mixed articles based on the person that writes the translation

Comment (itsmatteomanf) [reply docs/src/content/docs/it/guides/sidebar.mdx]: Ok, agreed. And committed.

**Pull request merged**